packages/devextreme/testing/tests/DevExpress.ui.widgets.pivotGrid/store.remote.tests.js

```diff
@@ -1594,7 +1594,7 @@ QUnit.module('Expanding items', moduleConfig, () => {
             };
             loadOptions[expandedPathOptionName] = [[2010], [2010, 2010, 1]];
             store.load(loadOptions).done(function() {
-                assert.deepEqual(actualFilter, [[['date1', '=', 2010]], 'and', [['date2', '=', 2010]]], 'rows count');
+                assert.deepEqual(actualFilter, [['date1', '=', 2010], 'and', ['date2', '=', 2010]], 'rows count');
             });
         });
```

Comment:
Same as this comment: https://github.com/DevExpress/DevExtreme/pull/28884/files#r1941724159